packages/teleport/src/services/localStorage/localStorage.ts

```diff
@@ -24,47 +23,6 @@ const storage = {
     window.localStorage.clear();
   },
 
-  subscribe(fn) {
```

Comment:
With all this removed I question the usefulness of this file for getting and setting a single key. 

It looks like we interact with `localStorage` directly in most places, in interest of consistency throughout the application those should maybe be all moved into this file or this file removed. But that's probably out of scope of this PR, thoughts on removing this in favour of an update?